frontend/views/containers/proposals/Mincome.vue

```diff
@@ -118,8 +118,9 @@ export default {
 
           this.ephemeral.currentStep += 1 // Show Success step
         } catch (e) {
-          console.error(`Failed to change mincome to ${mincomeAmount}`, e.message)
-          this.$refs.formMsg.danger(L('Failed to change mincome. {codeError}', { codeError: e.message }))
+          this.dumbError.dumbError = 'forcer-error' // DELETE BEFORE MERGE!!
+          console.error('Mincome submit() error:', mincomeAmount, e.message)
+          this.$refs.formMsg.danger(L('Failed to change mincome. {reportError}', LError(e)))
```

Comment:
Important rule I learned the hard way: to save Future Sandrina, make all user-facing errors unique.

So here it should be: `Failed to propose mincome change: {reportError}` (or, if you don't like the link, leave out `{reportError}` and replace it with `{errorMsg}` and set that to `e.message`

This way the error will be unique from the one below.

Reply:
I didn't understand how this solves the current problem. And I never said I "don't like" the link. There are situations where the link is needed and others where it isn't. We can't simply exclude it from every cases.

Reply:
Oh, that's not exactly what I'm talking about here (only partly).

We have keep the link, that's fine, the main issue I'm concerned about here is that the two error messages are the same, so a user comes to us and tells us, "The app said 'Failed to change mincome.'", we won't be able to tell where the error occurred.

This is why you always make the wording of error messages unique if they're located in different locations. It's a "people hack" that's really helpful. :)

So can you rewrite this as `Failed to propose mincome change: {reportError}`?

Reply:
Ah got it now, completely agree!